cluster/examples/kubernetes/ceph/csi/template/cephfs/csi-cephfsplugin-provisioner-dep.yaml

```diff
@@ -34,6 +34,21 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /csi
+        - name: csi-snapshotter
```

Comment:
I believe we need to conditionally enable the snapshotter if the k8s version supports it, otherwise the container logs for the snapshotter will spew errors. We also have this issue for the rbd snapshotter. However, I don't see an issue opened for it yet.

Reply:
Yeah. that would be good and can both ( RBD and CephFS ) handled together in a followup PR :+1:

Reply:
is it possible to do it before the release?

Reply:
we can defer it from this release and get it done for next patch release.

Reply:
Can we get this early next week?